### ISSUE TITLE
Column alignment switches between rows

### ISSUE THREAD

**Issue by dietrichson** [haozhu233/kableExtra#603]
**Describe the bug**
Column alignment different for each row when using multi-line text in columns. 
See the pdf for output: 
[testing-and-debug.pdf](https://github.com/haozhu233/kableExtra/files/6285578/testing-and-debug.pdf)

**To Reproduce**
Think about using `reprex` to generate an example in R. 

```
knitr::opts_chunk$set(echo = TRUE)
library(tidyverse)
library(kableExtra)
dt_lb <- data.frame(
  I = c(   "1\nStuff to do\n more stuff",
           "2\nStuff to do\n more stuff",
           "3\nStuff to do\n more stuff",
           "4\nStuff to do\n more stuff"),
  II =  c("1\nMore meetings\n and coffe-breaks",
          "2\nMore meetings\n and coffe-breaks",
          "3\nMore meetings\n and coffe-breaks",
          "4\nMore meetings\n and coffe-breaks"),
  III = c("1\nthis is \n more text",
          "2\nthis is \n more text",
          "3\nthis is \n more text",
          "4\nthis is \n more text")
)

dt_lb %>%
  mutate_all(linebreak) %>%
  kbl(
    format = "latex",
  ,booktabs = T, escape = F, 
  col.names = c("One","TWO","THREE")
  ).
```

**Comment by dietrichson** [comment]
This happens because you didn't specify alignment in the linebreak call.

Not quite sure what this means, should I add additional parameters in the
line:
> mutate_all(linebreak) %>%

Or am I missing a step?

**Comment by dmurdoch** [comment]
I think you just add extra parameters, e.g. `mutate_all(linebreak, align = "l") %.>% ...` .

**Comment by dmurdoch** [comment]
Fixed in #676
